server/src/main/java/org/opensearch/extensions/ExtensionsManager.java

```diff
@@ -567,6 +588,10 @@ public static String getRequestExtensionEnvironmentSettings() {
         return REQUEST_EXTENSION_ENVIRONMENT_SETTINGS;
     }
 
+    public static String getRequestExtensionDependencyInformation() {
+        return REQUEST_EXTENSION_DEPENDENCY_INFORMATION;
```

Comment:
Are we returning a string here? I see this method is not invoked anywhere, do we need it?

Reply:
I agree - this method isn't necessary since the variable is public